ArjunaJTS/interop/pom.xml

```diff
@@ -17,19 +17,18 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.narayana.quickstart.jts</groupId>
+    <artifactId>narayana-quickstarts-jts</artifactId>
+    <version>5.13.1.Final-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
```

Comment:
IIRC, you told me that `../pom.xml` is the default value, isn't it?

Reply:
Thanks @jmfinelli ! Yes, it is the default. I think I copied it from other projects. I will remove that line.